src/create-card.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+
+export default class CardCreator extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      question: '',
+      answer: ''
+    }
+    this.handleChange = this.handleChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+  handleChange(e) {
+    if (e.target.name === 'question') {
+      this.setState({
+        question: e.target.value
+      })
+    }
+    if (e.target.name === 'answer') {
+      this.setState({
+        answer: e.target.value
+      })
+    }
+  }
+  handleSubmit(e) {
+    e.preventDefault()
+    this.setState({question: '', answer: ''})
+    console.log(this.state)
```

Comment:
Here is where you want to call `this.props.onSubmit` (or something like that). The end goal for this feature is to put new flash cards into the state of the `App` component.